apps/web/assets/brunch-config.js

```diff
@@ -26,7 +26,10 @@ exports.config = {
   // Configure your plugins
   plugins: {
     babel: {
-      presets: ['latest', 'react']
+      presets: ['latest', 'react'],
+      plugins: [
+        "transform-class-properties"
```

Comment:
For my own edification could you tell me what this does?  Also, should we use single quotes for consistency or is double necessary here?

Reply:
Oh, nope. You can use single quotes for sure. 

This adds a specific Babel plugin for supporting the instance methods we were talking about earlier. Babel is actually just a big bundle of plugins, each of which address a certain JS feature you want to transpile.

Presets are just bundles of commonly used plugins (for the sake of convenience), but sometimes you have to pull in specific plugins as well

Reply:
Oh, nope. You can use single quotes for sure. 

This adds a specific Babel plugin for supporting the instance methods we were talking about earlier. Babel is actually just a big bundle of plugins, each of which address a certain JS feature you want to transpile.

Presets are just bundles of commonly used plugins (for the sake of convenience), but sometimes you have to pull in specific plugins as well

Reply:
Thanks @cafreeman 😁